.taskcat.yml

```diff
@@ -17,19 +17,21 @@ project:
   - us-west-2
   s3_bucket: ''
 tests:
-  BB-5:
+  bitbucket:
     parameters:
-      CidrBlock: 10.0.0.0/0
-      BitbucketVersion: 6.6.0
+      AvailabilityZones: $[taskcat_genaz_2]
+      BitbucketVersion: "6.6.0"
+      DBMasterUserPassword: "f925dO1ry_"
```

Comment:
I think we can change the passwords back to auto-generated - `$[taskcat_genpass_10S]`

The set of special characters generated by taskcat and supported by our templates is same:
https://github.com/atlassian/quickstart-atlassian-bitbucket/blob/develop/templates/quickstart-bitbucket-dc.template.yaml#L366

https://github.com/aws-quickstart/taskcat/blob/87ced9db06fe1d9b73cec7e3b94564b7d5ae8928/taskcat/_template_params.py#L225